src/__tests__/useQuery-test.tsx

```diff
@@ -461,18 +461,9 @@ it('should ignore apollo errors by default in non-suspense mode', async () => {
 
   await flushEffectsAndWait();
 
-  expect(consoleLogMock).toBeCalledTimes(1);
-  expect(consoleLogMock.mock.calls[0][0]).toMatchInlineSnapshot(
-    `"Unhandled error"`
-  );
-  expect(consoleLogMock.mock.calls[0][1]).toMatchInlineSnapshot(
-    `"Network error: Simulating network error"`
-  );
-
-  // TODO: It should show error state.
   expect(container).toMatchInlineSnapshot(`
 <div>
-  Loading without suspense
+  Network error: Simulating network error
```

Comment:
We should rename this test to e. g. _"shouldn't ignore apollo errors in non-suspense mode"_.